samples/msal-browser-samples/VanillaJSTestApp2.0/app/instanceAware/authConfig.js

```diff
@@ -0,0 +1,60 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "3fba556e-5d4a-48e3-8e1a-fd57c12cb82e",
+        authority: "https://login.windows-ppe.net/common/"
+    },
+    cache: {
+        cacheLocation: "sessionStorage", // This configures where your cache will be stored
+        storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+    },
+    system: {
+        loggerOptions: {
+            loggerCallback: (level, message, containsPii) => {
+                if (containsPii) {	
+                    return;	
+                }
+                switch (level) {	
+                    case msal.LogLevel.Error:	
+                        console.error(message);	
+                        return;	
+                    case msal.LogLevel.Info:	
+                        console.info(message);	
+                        return;	
+                    case msal.LogLevel.Verbose:	
+                        console.debug(message);	
+                        return;	
+                    case msal.LogLevel.Warning:	
+                        console.warn(message);	
+                        return;	
+                }
+            }
+        }
+    }
+};
+
+// Add here scopes for id token to be used at MS Identity Platform endpoints.
+const loginRequest = {
+    scopes: ["User.Read"],
+    extraQueryParameters: {
+        "instance_aware": "true"
```

Comment:
Does this need its own sample? Can we write a short doc instead?

Reply:
Would prefer to have a sample that shows how to use the flow and the graph endpoints correctly. Happy to revisit if we have a lot of pushback on this.

Reply:
I think we should write a doc (or update an existing one) for this regardless. The sample may or may not become redundant at that point

Reply:
Will definitely update the docs to add the instance_aware flow